pages.txt

```diff
@@ -3,70 +3,71 @@ Idobata makes software development fun
 
 By developers for developers
 Lightweight, familiar UI inspired by IRC
-Integration with the tools that you use
+Integrates with the tools that you use
 Sign Up
 
-Free for good
+Free forever
 
+Why?
+There are tons of reasons to use Idobata.
 
-WHY?
-WHY IDOBATA
-We made Idobata to solve developer's worries about team development.
+Idobata solves a lot of common problems with developer chat tools. 
 
-It's painful to maintain server for IRC...
+It's painful to maintain server for IRC.
 
-I wish I could @username style mention with type-ahead...
+I wish I could autocomplete my @username-style mentions. 
 
-I want easy and adequate integration with GitHub, CI and so on...
+I want simple integration with GitHub, CI, etc... 
 
-How to see transversely all messages which I am interested in?
+I'd like to see every message thread I am following in at a glance.
 
-Developer's chat should be more programmable...
+Developers' chat should be more programmable.
 
-ORGANIZATION
-ONE ACCOUNT, MULTIPLE TEAMS
-In Idobata, you can belong to multiple organizations such as enterprises and communities. In Organization, you can create multiple rooms as many as you want.
+Organization
+One account, multiple teams
 
-You can invite an outside person as a guest to a room. A guest can only join the rooms which allow access from the guest.
+Idobota lets you create Organizations, different groupings of chat rooms.
 
-From the Timeline view, you can cross view each room across each organization.
+You can give your members access to all rooms in your Organization, or limit them to specific ones. 
```

Comment:
Hmm, I'm not sure why I capitalized it... perhaps because it felt like a "proper" noun (like a name, company, etc). But, now that you mention it, I don't think it's necessary...

It doesn't look strange to me, but it's OK having it in lowercase too.